storage/diskmetricstore_test.go

```diff
@@ -783,7 +806,7 @@ func checkMetricFamilies(dms *DiskMetricStore, expectedMFs ...*dto.MetricFamily)
 	for i, got := range gotMFsAsStrings {
 		expected := expectedMFsAsStrings[i]
 		if expected != got {
-			return fmt.Errorf("expected metric family %#v, got %#v", expected, got)
+			return fmt.Errorf("expected metric family '%s', got '%s'", expected, got)
```

Comment:
```suggestion
			return fmt.Errorf("expected metric family %q, got %q", expected, got)
```

Reply:
I know it's tempting to suggest that, but the `'%s'` is intentional as the double-quoted metric families look horrible. Their string representation contains a lot of double quotes on their own, so `%q` escapes them all. It looks like 30% backslashes in the error message...

Reply:
A `%q` that uses single quotes would be cool.